docs/Documentation/Events-List.md

```diff
@@ -176,25 +176,39 @@ Creates an explosion. It can make fire and destroy blocks. You can also define p
     explosion 0 1 4 100;64;-100;survival
     ```
 
-## Folder: `folder`
+## :material-folder-play: Run multiple events: `folder`
 
 **persistent**, **static**
 
-It's something like a container for multiple events. You can use it to clarify your code.
-It also features optional delay and period measured in seconds (you can use ticks or minutes if you add `ticks` or `minutes` argument).
-It is persistent for events marked as _persistent_, which means that the events will be fired even after the player logs out.
+This event wraps multiple events inside itself. Once triggered, it simply executes it's events.
+This is usefully to easily refer to a bunch of events at once, e.g. in a conversation.
+
+Events marked as _persistent_ will be fired even after the player logs out.
 Beware though, all conditions are false when the player is offline (even inverted ones),
 so those events should not be blocked by any conditions!
-The only required argument is a list of events separated by commas.
 
-There are also three optional arguments: `delay:`, `period:` and `random:`.
-Delay and Period is a number of seconds. Delay is the time before execution and period is the time between each event. It's optional and leaving it blank is the same as `delay:0` or `period:0`.
-Random is the amount of events, that will be randomly chosen to fire. It's optional and leaving it blank or omit it will fire all events.
 
-!!! example
-    ```YAML
-    folder event1,event2,event3 delay:5 period:5 random:1
-    ```
+| Parameter       | Syntax            | Default Value          | Explanation                                                                                     |
+|-----------------|-------------------|------------------------|-------------------------------------------------------------------------------------------------|
+| _events to run_ | eventName1,event2 | :octicons-x-circle-16: | One or multiple events to run. Contains event names seperated by commas.                        |
+| _delay_         | Keyword           | 1 tick                 | The delay before the folder starts executing it's events.                                       |
+| _period_        | period:number     | 1 tick                 | The time between each event. Minimum value is one tick (~20ms).                                 |
+| _time unit_     | Keyword           | Seconds                | The unit of time to use for delay and period. Either `ticks` or `minutes`. Omit to use seconds. |
+| _random_        | random:number     | Disabled               | Enables "random mode". Will randomly pick the defined amount of events .                        |
```

Comment:
I find the syntax here quite confusing. Maybe this is better 🤔:
```suggestion
| Parameter       | Example Syntax      | Default Value          | Explanation                                                                                     |
|-----------------|---------------------|------------------------|-------------------------------------------------------------------------------------------------|
| _events to run_ | `eventName1,event2` | :octicons-x-circle-16: | One or multiple events to run. Contains event names seperated by commas.                        |
| _delay_         | `delay`             | 1 tick                 | The delay before the folder starts executing it's events.                                       |
| _period_        | `period:10`         | 1 tick                 | The time between each event. Minimum value is one tick (~20ms).                                 |
| _time unit_     | `ticks` / `minutes` | Seconds                | The unit of time to use for delay and period. Either `ticks` or `minutes`. Omit to use seconds. |
| _random_        | `random:3`          | Disabled               | Enables "random mode". Will randomly pick the defined amount of events .                        |
```

Reply:
We discussed this, this can maybe be improved in general, but this is not part of this PR